### PR TITLE
Depend on separate libuv package

### PR DESCRIPTION
**Pull request by halter73**
#730

### CODE REVIEW

File: samples/SampleApp/project.json [hunk 1]
```diff
@@ -1,18 +1,6 @@
 {
   "version": "1.0.0-*",
   "dependencies": {
-    "Microsoft.AspNetCore.Internal.libuv-Darwin": {
```
Comment (cesarbs) [inline samples/SampleApp/project.json]: Is this the only repo with such kind of dependencies?

**Comment by cesarbs** [comment]
Sweet! :+1:

**Comment by cesarbs** [comment]
:shipit: I guess, or wait until Libuv package available so AppVeyor and Travis pass?

File: samples/SampleApp/project.json (hunk 1 above)
Comment (halter73) [inline samples/SampleApp/project.json]: Yes. It's only required if you take a project reference to Kestrel.

**Comment by cesarbs** [comment]
Why are Travis and AppVeyor failing?

**Pull request closed without merging**